docs/current_docs/quickstart/183917-daggerverse.mdx

```diff
@@ -0,0 +1,18 @@
+---
+slug: /quickstart/183917/daggerverse
+hide_table_of_contents: true
+title: "Use modules from the Daggerverse"
+---
+
+# Quickstart
+
+## Use modules from the Daggerverse
+
+As you've seen in previous sections, modules can be consumed from remote sources like GitHub, or installed and used locally. The [Daggerverse](https://daggerverse.dev) is an online catalog of Dagger modules that enables users to easily discover and use modules created by the Dagger community.
+
+All Dagger CLI commands work as expected with Daggerverse URLs. Here are a few more examples for you to try:
+
+- [Publish a container to the ttlsh registry](https://daggerverse.dev/mod/github.com/shykes/daggerverse/ttlsh): `dagger -m github.com/shykes/daggerverse/ttlsh call publish --ctr=alpine:latest`
+- [Play a game of yamlinvaders](https://daggerverse.dev/mod/github.com/sipsma/daggerverse/yamlinvaders): `dagger -m github.com/sipsma/daggerverse/yamlinvaders call play terminal`
+- [Stitch videos together](https://daggerverse.dev/mod/github.com/marcosnils/daggerverse/videostitch): `wget -q https://download.samplelib.com/mp4/sample-5s.mp4 && wget -q https://download.samplelib.com/mp4/sample-10s.mp4 && dagger -m github.com/marcosnils/daggerverse/videostitch@main call -o output.mp4 stitch --src-dir=.`
+- [Scan a container image for vulnerabilities](https://daggerverse.dev/mod/github.com/jpadams/daggerverse/trivy): `dagger -m github.com/jpadams/daggerverse/trivy scan-image call --image-ref=alpine/git:latest`
```

Comment:
```suggestion
- [Scan a container image for vulnerabilities](https://daggerverse.dev/mod/github.com/jpadams/daggerverse/trivy)
- [Stitch videos together](https://daggerverse.dev/mod/github.com/marcosnils/daggerverse/videostitch)
- [Build a Go project](https://daggerverse.dev/mod/github.com/kpenfound/dagger-modules/golang)
- [Play a game of yamlinvaders](https://daggerverse.dev/mod/github.com/sipsma/daggerverse/yamlinvaders)
- [Generate GIFs of interactive terminal sessions](https://daggerverse.dev/mod/github.com/shykes/daggerverse/termcast)
- [Integrate with Terraform](https://daggerverse.dev/mod/github.com/Excoriate/daggerverse/terraform)
```

Reply:
@shykes instead of having only links as suggested above, I included an example `dagger call` command for each. I thought this would get a user engaged immediately, by being able to try a module without needing to refer to its docs first. Was there a specific reason you removed those? If compat is a concern, I can add version numbers to each command.

Reply:
> @shykes instead of having only links as suggested above, I included an example `dagger call` command for each. I thought this would get a user engaged immediately, by being able to try a module without needing to refer to its docs first. Was there a specific reason you removed those? If compat is a concern, I can add version numbers to each command.

IMO the `dagger call` examples don't show how to use the daggerverse; they simply show new variations of what we've already showed in previous sections: how to call a function once you know its module's URL.

In this section we show how to search for modules in the daggerverse: linking to a few modules pages, to show results of possible searches, accomplishes that.

> try a module without needing to refer to its docs first

In this section, clicking through to the docs is the point.

One possibility, is to start this section with an intro paragrap that shows *just the `dagger call` commands*. The paragraph says "Dagger's superpower is its community. All around the world, Daggernauts are encoding their expertise into functions, and sharing them for anyone to reuse. (examples). But how to the perfect function for the job? (transition to next section: the daggerverse, how to search, examples of links, etc)

Reply:
I like the second option, added it now. Please let me know your thoughts @shykes, or approve if OK - thank you!